sql/database.sql

```diff
@@ -242,35 +242,37 @@ CREATE TABLE `categories` (
-INSERT INTO `categories` VALUES (27, 'Onsite Portal', '', 1, 51, 56, 'patients|docs');
-INSERT INTO `categories` VALUES (28, 'Patient', '', 27, 52, 53, 'patients|docs');
-INSERT INTO `categories` VALUES (29, 'Reviewed', '', 27, 54, 55, 'patients|docs');
+INSERT INTO `categories` VALUES (1, 'Categories', '', 0, 0, 59, 'patients|docs');
+INSERT INTO `categories` VALUES (2, 'Lab Report', '', 1, 1, 4, 'patients|docs');
+INSERT INTO `categories` VALUES (3, 'Imaging', '', 2, 2, 3, 'patients|docs');
+INSERT INTO `categories` VALUES (4, 'Medical Record', '', 1, 5, 6, 'patients|docs');
+INSERT INTO `categories` VALUES (5, 'Patient Information', '', 1, 7, 12, 'patients|docs');
+INSERT INTO `categories` VALUES (6, 'Patient ID card', '', 4, 8, 9, 'patients|docs');
+INSERT INTO `categories` VALUES (7, 'Advance Directive', '', 1, 13, 20, 'patients|docs');
+INSERT INTO `categories` VALUES (8, 'Do Not Resuscitate Order', '', 6, 14, 15, 'patients|docs');
+INSERT INTO `categories` VALUES (9, 'Durable Power of Attorney', '', 6, 16, 17, 'patients|docs');
+INSERT INTO `categories` VALUES (10, 'Living Will', '', 6, 18, 19, 'patients|docs');
+INSERT INTO `categories` VALUES (11, 'Patient Photograph', '', 4, 10, 11, 'patients|docs');
+INSERT INTO `categories` VALUES (12, 'CCR', '', 1, 21, 22, 'patients|docs');
+INSERT INTO `categories` VALUES (13, 'CCD', '', 1, 23, 24, 'patients|docs');
+INSERT INTO `categories` VALUES (14, 'CCDA', '', 1, 25, 26, 'patients|docs');
+INSERT INTO `categories` VALUES (15, 'Eye Module', '', 1, 27, 52, 'patients|docs');
+INSERT INTO `categories` VALUES (16, 'Communication - Eye', '', 14, 28, 29, 'patients|docs');
+INSERT INTO `categories` VALUES (17, 'Encounters - Eye', '', 14, 30, 31, 'patients|docs');
+INSERT INTO `categories` VALUES (18, 'Imaging - Eye', '', 14, 32, 51, 'patients|docs');
+INSERT INTO `categories` VALUES (19, 'OCT - Eye', 'POSTSEG', 17, 33, 34, 'patients|docs');
+INSERT INTO `categories` VALUES (20, 'FA/ICG - Eye', 'POSTSEG', 17, 35, 36, 'patients|docs');
+INSERT INTO `categories` VALUES (21, 'External Photos - Eye', 'EXT', 17, 37, 38, 'patients|docs');
+INSERT INTO `categories` VALUES (22, 'AntSeg Photos - Eye', 'ANTSEG', 17, 39, 40, 'patients|docs');
+INSERT INTO `categories` VALUES (23, 'Optic Disc - Eye', 'POSTSEG', 17, 41, 42, 'patients|docs');
+INSERT INTO `categories` VALUES (24, 'Fundus - Eye', 'POSTSEG', 17, 43, 44, 'patients|docs');
+INSERT INTO `categories` VALUES (25, 'Radiology - Eye', 'NEURO', 17, 45, 46, 'patients|docs');
+INSERT INTO `categories` VALUES (26, 'VF - Eye', 'NEURO', 17, 47, 48, 'patients|docs');
+INSERT INTO `categories` VALUES (27, 'Drawings - Eye', '', 17, 49, 50, 'patients|docs');
+INSERT INTO `categories` VALUES (28, 'Onsite Portal', '', 1, 53, 58, 'patients|docs');
+INSERT INTO `categories` VALUES (29, 'Patient', '', 27, 54, 55, 'patients|docs');
+INSERT INTO `categories` VALUES (30, 'Reviewed', '', 27, 56, 57, 'patients|docs');
```

Comment:
oh goodness, we get to figure out this again :)
(inserting stuff is a bit complicated, especially in the upgrade script, but it can be done)
Here's an example in the upgrade script:
https://github.com/openemr/openemr/blob/master/sql/5_0_0-to-5_0_1_upgrade.sql#L402-L408

Reply:
Awesome, thanks for the tip. I'll transfer the INSERT and UPDATE statements to `5_0_0-to-5_0_1_upgrade.sql`. I was able to figure it out but didn't know where to put them.